_posts/2020-03-02-tomcat-thread-model.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:    post
+title:     "Tomcat - model przetwarzania żądań"
+date:      2020-03-19 10:00:00 +0100
+published: false
+author:    jwilczewski
+tags:
+    - java
+    - tomcat
+    - nio
+    - connector
+---
+
+Tomcat jest jednym z najpopularniejszych serwerów webowych dla aplikacji pisanych w Javie. Jest podstawowym kontenerem aplikacji springboot'owych. Tworząc nowy projekt często polegamy na jego domyślnej konfiguracji. Kiedy projekt dojrzewa do wdrożenia produkcyjnego i musi zmierzyć się, z obsługą dużego ruchu często konieczne okazuje się dostrojenie tej konfiguracji. W tym artykule skupię się na konfiguracji connectorów na przykładzie pewnego problemu produkcyjnego. Opis konfiguracji Tomcata bazuje na wersji 7.x, ale jest w zasadzie aktualny również dla wyższych wersji.
```

Comment:
Jak coś to moja uwaga jest w stylu "nie znam się ale się wypowiem" :D,
Podczas czytania, wydaje mi się że lepiej by się czytało jakby przecinek był w inny miejscu (ten po `i musi zmierzyć się`), czyli:

`i musi zmierzyć się z obsługą dużego ruchu, często konieczne okazuje się dostrojenie tej konfiguracji.`

ewentualnie jeszcze jeden dodatkowy przecinek właśnie w tym miejscu. Ale najlepiej żeby ktoś jeszcze się wypowiedział na ten temat :)

Reply:
Rzeczywiście fuj. Poprawiłem.